server/server.go

```diff
@@ -68,6 +69,12 @@ type Info struct {
 	// Route Specific
 	Import *SubjectPermission `json:"import,omitempty"`
 	Export *SubjectPermission `json:"export,omitempty"`
+
+	// Gateways Specific
+	Gateway     string   `json:"gateway,omitempty"`      // Name of the origin Gateway (sent by gateway's INFO)
```

Comment:
This is just cluster name IMO.

Reply:
Originally we discussed that gateway may be run without cluster. From a configuration perspective, we need a different port to interpret TCP connections as either a client, route or gateway. So it would be strange to have cluster{} have a name and gateway{} not, especially if gateway can exist without cluster?